tests/common/test_telemetryevent.py

```diff
@@ -19,23 +19,23 @@
 from tests.lib.tools import AgentTestCase
 
 
-def get_test_event(name="DummyExtension", op="Unknown", is_success=True, duration=0, version="foo", evt_type="", is_internal=False,
```

Comment:
nose was identifying this as a test; renamed it and moved it to the TestTelemetryEvent class